.github/workflows/nix-dev-cache.yaml

```
@@ -24,6 +24,11 @@ jobs:
           - macOS-14
     steps:
     - uses: actions/checkout@v4
+    - name: set up nix mount points on Linux
```

```suggestion
    - name: mount Nix store on larger partition
```
To give more of a reason why.